subt/check_phone.py

```diff
@@ -0,0 +1,30 @@
+from osgar.logger import LogReader, lookup_stream_id
+from osgar.lib.serialize import deserialize
+
+CELL_PREFIX = "PhoneArtifact"
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Convert logfile to AVI video')
+    parser.add_argument('logfile', help='recorded log file')
+    args = parser.parse_args()
+
+    only_stream = lookup_stream_id(args.logfile, 'wifi.wifiscan')
+    best = None
+    with LogReader(args.logfile, only_stream_id=only_stream) as log:
+        for timestamp, stream_id, data in log:
+            wifi_list = deserialize(data)
+            for name, signal in wifi_list:
+                if name.startswith(CELL_PREFIX):
+                    print(timestamp, name, signal)
+                    if best is None or abs(best[2]) > abs(signal):
```

Comment:
This is a bit non-obvious. Maybe `best[2] < signal` would be more understandable?

Reply:
I rewrote with swapped parameters ... I am not sure, what is easier to understand - we failed on this inequality 3rd day, so I am not sure.

Reply:
Adding the `abs` makes it confusing for me since it turn the inequality upside down. We are looking for the largest number, right? They just happen to be negative... which does not change anything. Still looking for the largest number.